x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -85,6 +86,11 @@ const TabContentWrapper = styled.div`
   position: relative;
 `;
 
+const StyledNotificationBadge = styled(EuiNotificationBadge)`
+  margin-top: -1px;
```

Comment:
This feels a little hacky 😉 . Could this maybe be accomplished with flexbox? What are you trying to do/fix here?

Reply:
NotificationBadge appeared a bit squished from top and this seemed to be the smallest fix - will attempt to fix more gracefully